common/src/TrenchBroomApp.cpp

```diff
@@ -613,7 +613,7 @@ void TrenchBroomApp::openFilesOrWelcomeFrame(const QStringList& fileNames)
     for (const auto& fileName : fileNames)
     {
       const auto path = IO::pathFromQString(fileName);
-      anyDocumentOpened = anyDocumentOpened | (!path.empty() && openDocument(path));
+      anyDocumentOpened = anyDocumentOpened || (!path.empty() && openDocument(path));
```

Comment:
This would change the behavior to open only the first successful file, due to the short-circuiting behavior of `||`.

That doesn't *seem* like the desired behavior based on the name of the function (I would infer `openFiles` to mean open *all* files, not open *any* file)

Reply:
Ah thanks, that went right past me.

Reply:
Should be fixed in https://github.com/TrenchBroom/TrenchBroom/pull/4450